lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -4619,6 +4643,32 @@ SwiftASTContext::ReconstructType(ConstString mangled_typename) {
                                      "\" was not found");
 }
 
+llvm::Expected<swift::TypeBase *>
+SwiftASTContext::ReconstructType(ConstString mangled_typename) {
```

Comment:
Unless we need a ConstString later anyway, maybe this should now take a StringRef?

Reply:
A lot of the helper functions used in ReconstructType take a ConstString, so I don't think that's work changing right now.